src/database/models/reportRoleMapping.js

```diff
@@ -0,0 +1,45 @@
+'use strict'
+
+module.exports = (sequelize, DataTypes) => {
+	const ReportRoleMapping = sequelize.define(
+		'ReportRoleMapping',
+		{
```

Comment:
@sumanvpacewisdom make the same schema changes which mentioned for the migration.